main.tf

```diff
@@ -0,0 +1,191 @@
+/**
+* ## Module: cloudposse/terraform-aws-lambda-elasticsearch-cleanup
+*
+* This module creates a scheduled Lambda function which will delete old
+* Elasticsearch indexes using SigV4Auth authentication. The lambda 
+* function can optionally send output to an SNS topic if the topic ARN
+* is given
+*/
+
+# Terraform
+#--------------------------------------------------------------
+terraform {
+  required_version = ">= 0.10.2"
+}
+
+# Data
+#--------------------------------------------------------------
+data "aws_vpc" "default" {
```

Comment:
would not use unnecessary data lookups.
it could lead to some race conditions and other issues when the module is used in a complex module hierarchy.
better to provide the subnets IDs in a var.
for the VPC, probably better to not look it up, and not to use the CIDR block for the SG rules, but provide a VPC security group - at least we do it in many other modules